content/en/user-guide/aws/athena/index.md

```diff
@@ -177,7 +177,28 @@ The `SELECT` statement above currently requires us to prefix the database/table
 
 ## Iceberg Tables
 
-The LocalStack Athena implementation also supports [Iceberg tables](https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-creating-tables.html) - more details and samples will be provided here soon.
+The LocalStack Athena implementation also supports [Iceberg tables](https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-creating-tables.html). You can define an Iceberg table in Athena using the `CREATE TABLE` statement, as shown in the example below:
+
+```sql
+CREATE TABLE mytable (c1 integer, c2 string, c3 double)
+LOCATION 's3://mybucket/prefix/' TBLPROPERTIES ( 'table_type' = 'ICEBERG' )
+```
+
+Once the table has been created and data inserted into it, you can see the Iceberg metadata and data files being created in S3:
+
+```bash
```

Comment:
nit: This isn't a bash script, so I think this language config should be removed.